### PR TITLE
python3Packages.pyroon: init at 0.0.37

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Python library to interface with the Roon API

https://github.com/pavoni/pyroon

This is a Home Assistant dependency.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 124704` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python38Packages.pyroon</li>
    <li>python39Packages.pyroon</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 124704` at d04ee258 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built successfully:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python38Packages.pyroon</li>
    <li>python39Packages.pyroon</li>
  </ul>
</details>

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 124704 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>3 packages built:</summary>  <ul>    <li>home-assistant</li>    <li>python38Packages.pyroon</li>    <li>python39Packages.pyroon</li>  </ul></details>

File: pkgs/servers/home-assistant/component-packages.nix [hunk 1]
```diff
@@ -701,7 +701,7 @@
     "rocketchat" = ps: with ps; [ ]; # missing inputs: rocketchat-API
     "roku" = ps: with ps; [ rokuecp ];
     "roomba" = ps: with ps; [ roombapy ];
-    "roon" = ps: with ps; [ ]; # missing inputs: roonapi
+    "roon" = ps: with ps; [ pyroon ];
```
Comment (mweinelt) [inline pkgs/servers/home-assistant/component-packages.nix:704]: Did you set that value manually? Because it is wrong.

**Pull request merged**